In d4737a39ca8702e32ad0a47941f93fef2872966f SSL loading automatically was
disabled for general use, but we want to have it in the Debian package.

--- a/src/hooks.lisp
+++ b/src/hooks.lisp
@@ -30,10 +30,8 @@
     ;; handles some context and things around loading with CFFI.
     (cl+ssl:reload)))
 
-#|
 #+ccl  (push #'open-foreign-libs *lisp-startup-functions*)
 #+sbcl (push #'open-foreign-libs sb-ext:*init-hooks*)
-|#
 
 #+ccl  (push #'close-foreign-libs *save-exit-functions*)
 #+sbcl (push #'close-foreign-libs sb-ext:*save-hooks*)
